miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/StatsController.java

```diff
@@ -244,16 +227,11 @@ public ModelAndView ls454Stats(@PathVariable(value = "referenceId") Long referen
       }
       try {
         SequencerInterrogator context = SequencerInterrogatorFactory.getSequencerInterrogator(sr);
-        Status status = context.getRunStatus(runName);
-        if (status != null) {
-          model.put(ModelKeys.SEQUENCER_NAME.getKey(), sr.getName());
-          model.put(ModelKeys.SEQUENCER_ID.getKey(), sr.getId());
-          model.put(ModelKeys.RUN_ID.getKey(), requestManager.getRunByAlias(runName).getId());
-          model.put(ModelKeys.RUN_NAME.getKey(), runName);
-          model.put(ModelKeys.RUN_STATUS.getKey(), status);
-        } else {
-          model.put(ModelKeys.ERROR.getKey(), MisoWebUtils.generateErrorDivMessage("Cannot consume the xml file for the given run name: " + runName));
-        }
+        model.put(ModelKeys.SEQUENCER_NAME.getKey(), sr.getName());
+        model.put(ModelKeys.SEQUENCER_ID.getKey(), sr.getId());
+        model.put(ModelKeys.RUN_ID.getKey(), requestManager.getRunByAlias(runName).getId());
+        model.put(ModelKeys.RUN_NAME.getKey(), runName);
+        model.put(ModelKeys.RUN_STATUS.getKey(), sr);
```

Comment:
This model attribute is for the xml. Can be commented out or removed since it'll be replaced, but don't add something different in its place. There is a `ModelKey.SEQUENCER` if the sequencer is required